lib/improver/psychrometric_calculations/psychrometric_plugins.py

```diff
@@ -0,0 +1,96 @@
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Module to contain plugins for use with psychrometric calculations"""
+
+import iris
+
+
+from improver.psychrometric_calculations.psychrometric_calculations import (
+    wet_bulb)
+
+
+class WetBulb(object):
+    """Calculate the wet bulb temperature given cubes of temperature,
+    relative humidity and pressure"""
+
+    def __init__(self):
+        """Initialise class."""
+        pass
+
+    def __repr__(self):
+        """Represent the configured plugin instance as a string."""
+        result = ('<WetBulbTemperature>')
+        return result
+
+    @staticmethod
+    def process(temperature, rel_humidity, pressure):
+        """Calculate the Wet Bulb Temperature
+
+        Args:
+            temperature : iris.cube.Cube
+                Cube of temperature
+            rel_humidity : iris.cube.Cube
+                Cube of relative humidity
+            pressure : iris.cube.Cube
+                Cube of pressure
+
+        Returns:
+            wet_bulb : iris.cube.Cube
+                Cube containing the wet bulb temperature
+
+        Raises:
+            TypeError: If any of the inputs are not cubes.
+            ValueError: If any of the input cubes are not the same shape.
+        """
+        # Check that inputs are of correct type
+        if not isinstance(temperature, iris.cube.Cube):
+            emsg = "Temperature is not a cube, but {}"
+            raise TypeError(emsg.format(type(temperature)))
+        if not isinstance(rel_humidity, iris.cube.Cube):
+            emsg = "Temperature is not a cube, but {}"
+            raise TypeError(emsg.format(type(rel_humidity)))
+        if not isinstance(pressure, iris.cube.Cube):
+            emsg = "Pressure is not a cube, but {}"
+            raise TypeError(emsg.format(type(pressure)))
+
+        # Check that cubes are all the same shape
```

Comment:
Might want to use utility fn here? Besides if this is wrong you will get a sensible broadcast/shape error from IRIS, so is it needed at all?